src/sprites/player.js

```diff
@@ -0,0 +1,17 @@
+import Phaser from 'phaser';
+
+export default class extends Phaser.Sprite {
```

Comment:
It's a good practice to give your `class` a name. It's helpful when debugging and looking at error stack traces. e.g.:

```js
export default class PlayerSprite extends Phaser.Sprite {
```
-------------------------------^